classes/Kohana/Asset/Collection.php

```diff
@@ -38,6 +38,30 @@
 	 * @var  int  last modified time
 	 */
 	protected $_last_modified = NULL;
+	
+	/**
+	 * @var bool show merged files paths
+	 */
+	protected $_show_paths = TRUE;
+	
+	/**
+	 * Setter / Getter for merged files path displaying
+	 * @author Piotr Gołasz <pgolasz@gmail.com>
```

Comment:
I'm sorry, but could you remove the `@author` annotation as it doesn't bring much value, such PHPDoc becomes stale and Git blame is better anyway. We need to remove it from other parts of the source code as well.